templates/zerver/integrations/capistrano.md

```diff
@@ -14,26 +18,24 @@ Get Zulip notifications for your Capistrano deploys!
           desc "Post a message to Zulip after deploy"
           task :humbug do
             run_locally "echo 'I just deployed to #{stage}! :tada:' | zulip-send \
-            --user capistrano-bot@example.com --api-key a0b1c2d3e4f5a6b7c8d9e0f1a2b3c4d5 \
+            --user <email of bot> --api-key a0b1c2d3e4f5a6b7c8d9e0f1a2b3c4d5 \
             --site={{ api_url }} \
             --stream commits --subject deployments || true"
           end
         end
 
-    The `--user` and `--api-key` should be the email and API key of the Zulip
-    bot created above. You can also put these values in a `~/.zuliprc` file on
-    your Capistrano machine. See our [API docs](/api/) for instructions on
-    creating that file.
+{end_tabs}
 
-!!! tip ""
+{!congrats.md!}
 
-    You can change the `deploy` above to another step of
```

Comment:
Split this into a point in the Configuration Options section and the link in the Related Documentation section.